source/unified-test-format/tests/invalid/kmsProviders-invalid_provider.yml

```
@@ -0,0 +1,18 @@
+description: kmsProviders-invalid_provider
```

Alternatively: `clientEncryptionOpts-kmsProviders-invalid_provider`, but I think this is more consistent with the existing filenames since it's testing the `additionalProperties` rule specifically.

```suggestion
description: clientEncryptionOpts-kmsProviders-additionalProperties
```